src/BottomNavigation/BottomNavigationButton.js

```diff
@@ -114,10 +114,11 @@ export default class BottomNavigationButton extends Component {
       [classes.selected]: selected,
       [classes.selectedIconOnly]: !showLabelProp && !selected,
     }, classNameProp);
-    const iconClassName = classNames(classes.icon,
-      isValidElement(iconProp) ? iconProp.props.className : null);
+    const iconClassName = classNames(classes.icon, isValidElement(iconProp) ?
+      iconProp.props.className :
```

Comment:
I'm not sure to understand. What the motivation for moving the icon `className` to the parent `div`?